site/docs/v1/tech/apis/forms.adoc

```diff
@@ -0,0 +1,155 @@
+---
```

Comment:
I've been trying to get better about the response code doc. For example, this API will not return a `503`, so there are some arguments you can pass to the response code macro so that it omits `503`. 

You can check each method response code doc to see how they are returned.